libs/utils/decorate.js

```diff
@@ -117,6 +117,29 @@ export const decorateBlockHrs = (el) => {
   if (hasHr && els.length) el.classList.add('has-divider');
 };
 
+export const decorateBgContent = (el) => {
```

Comment:
do we need the flexibility for authors to choose the color? maybe we can just offer a few types of background (tied to classes) and save the js processing?